docker-compose.yml

```diff
@@ -95,6 +95,23 @@ services:
     networks:
       - backend-network
 
+  rabbitmq:
+    image: "rabbitmq:3-management"
+    container_name: rabbitmq
+    ports:
+      - "5672:5672"
+      - "15672:15672" # Management UI
+    networks:
```

Comment:
If ur using env.RABBITMQ_URL please add the environment here, plz refer to other existing services